README.md

```diff
@@ -1,2 +1,30 @@
-# piccolo_docker
+## piccolo_docker
 Easily spin up databases for your Piccolo project using Docker
+
+### Requirements
+- [Install poetry](https://python-poetry.org/docs/#installation)
+- [Install docker](https://docs.docker.com/get-docker/)
+
+### Setup
+
+- Clone this repo
+- Open `.env`
+- Add database name
+- Add password
+
```

Comment:
We just need to add some docs about registering the app with Piccolo's `AppRegistry`, so it's added to the Piccolo CLI.

Reply:
This is already inside piccolo_conf.py, unless you mean something else.

Reply:
@antonio-one Yeah, sorry - I just meant in the README file it could say something like:

```markdown
To add use this app to your Piccolo project, add it to your `AppRegistry` in `piccolo_conf.py`.

APP_REGISTRY = AppRegistry(apps=["piccolo_docker.user.piccolo_app"])
```

Reply:
Done